libraries/chain/steem_evaluator.cpp

```diff
@@ -2506,14 +2505,18 @@ void delegate_vesting_shares(database& _db, const chain_properties& median_props
                 GOLOS_THROW_MISSING_OBJECT("vesting_delegation_object", op.delegator);
```

Comment:
`op.delegator` does not fully describe missing object. Full key contains both delegator and delegatee